qemu/tests/cfg/balloon_check.cfg

```diff
@@ -9,7 +9,10 @@
     free_mem_cmd = cat /proc/meminfo |grep MemFree
     variants:
         - balloon_base:
-
```

Comment:
Do we need this blank?

Reply:
@vivianQizhu No, it can work well without the blank.